src/lib.rs

```diff
@@ -36,8 +46,10 @@
 //! which follows a build pattern. The builders are finished using:
 //!
 //! * [`.call()`](struct.Request.html#method.call) without a request body.
-//! * [`.send()`](struct.Request.html#method.send) with a request body as `Read`.
+//! * [`.send()`](struct.Request.html#method.send) with a request body as `Read` (chunked encoding).
```

Comment:
If you set a `content-length` header manually it doesn't do chunked, but maybe that's too much detail.